lib/Transforms/Scalar/BackedgeCutter.cc

```diff
@@ -47,6 +47,24 @@ char BackedgeCutter::ID = 0;
 
 } // namespace
 
+static void createAssert(BranchInst &TI, Function &F, BasicBlock &dst,
+                         SeaBuiltinsInfo &SBI) {
+  // insert verifier.assert{.not} function
+  auto *assertFn =
+      SBI.mkSeaBuiltinFn(TI.getSuccessor(0) == &dst ? SeaBuiltinsOp::ASSERT_NOT
+                                                    : SeaBuiltinsOp::ASSERT,
+                         *F.getParent());
+  auto ci = CallInst::Create(assertFn, TI.getCondition(), "", &TI);
```

Comment:
what is condition for unconditional branch?

Reply:
Fixed.